### DISCUSSION TITLE
Adjustment of the authorization for the status 'User'

### DISCUSSION THREAD

**Discussion by DJ-SteffEN** [ampache/ampache#3972] [Q&A] [answered]
Hello,

I would like a user to be able to change only certain preferences.
He should not change the full name and email address.
At the moment a user ([inthemix.at](https://inthemix.at)) cannot change anything, the message 'Update failed' always appears.
A content or catalog manager, on the other hand, can change everything.
I would like to know if this is the default or where I can change it?

Thanks for your help
DJ Steffen

**Comment by DJ-SteffEN** [comment]
Hello,

I have in 'show_account.inc.php'
`$readonly = ($client->access > 25) ? '' : 'readonly';`
inserted
and the corresponding html inputs with
`<?php echo $readonly; ?>`
extended.
So the only question that remains is why only the user is not allowed to change anything.

Thanks in advance

**Comment by lachlan-00** [comment, marked as answer]
simple_user_mode would probably cover you here
![image](https://github.com/ampache/ampache/assets/1305249/8c5553bf-6847-4c00-bec1-68f461399f56)

This will block access to the account page unless you're an admin and blocks a few update functions so only the admin can change those things
![image](https://github.com/ampache/ampache/assets/1305249/c4782379-817d-46f0-9154-f7a58f080d7e)

**Comment by DJ-SteffEN** [comment]
Hello lachlan-00

thanks for the quick and detailed answer.
